files/setup-ssh.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
Could you rename it from generic `setup-ssh.py` to something like `setup-ssh-for-caasp`? 
We only need it for caasp and it's not deductible from file location so let's make it clear from the name...

Reply:
good point, I changed it to `setup-ssh-caasp-ci.sh`

Reply:
fixed?